uint8 agent_number
sensor_msgs/BatteryState battery
